extra/3-magic-8-ball.js

```diff
@@ -60,8 +86,49 @@ function shakeBall() {
 */
 function checkAnswer(answer) {
   //Write your code in here
+  let veryPositiveList = [
+    "It is certain.",
+    "It is decidedly so.",
+    "Without a doubt.",
+    "Yes - definitely.",
+    "You may rely on it.",
+  ];
+  let positiveList = [
+    "As I see it, yes.",
+    "Most likely.",
+    "Outlook good.",
+    "Yes.",
+    "Signs point to yes.",
+  ];
+  let negativeList = [
+    "Reply hazy, try again.",
+    "Ask again later.",
+    "Better not tell you now.",
+    "Cannot predict now.",
+    "Concentrate and ask again.",
+  ];
+  let veryNegativeList = [
+    "Don't count on it.",
+    "My reply is no.",
+    "My sources say no.",
+    "Outlook not so good.",
+    "Very doubtful.",
+  ];
+  let outputMessage;
+  if (veryPositiveList.includes(answer)) {
+    outputMessage = "very positive";
+  } else if (positiveList.includes(answer)) {
+    outputMessage = "positive";
+  } else if (negativeList.includes(answer)) {
+    outputMessage = "negative";
+  } else {
```

Comment:
What happens if the value of `answer` is not in the `veryNegativeList`? I think it would be better to make sure it is included:
```js
} else if (veryNegativeList.includes(answer)) {
    outputMessage = "very negative";
} else {
    outputMessage = "error";
}
```
or something like that.